### PR TITLE
Remove unnecessary cards in standalone app

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

#675 

## What does this implement/fix? Explain your changes.

Remove those duplicated cards
スクリーンショット 2024-01-02 11 56 11

### CODE REVIEW

File: standalone_app/src/components/PlotImportance.tsx [hunk 1]
```diff
@@ -59,17 +59,15 @@ export const PlotImportance: FC<{ study: Study }> = ({ study }) => {
...
-    </Card>
+    <Box>
+      <Typography
+        variant="h6"
+        sx={{ margin: "1em 0", fontWeight: theme.typography.fontWeightBold }}
+      >
+        Hyperparameter Importance
+      </Typography>
+      <Box id={plotDomId} sx={{ height: "450px" }} />
+    </Box>
```
Comment (c-bata) [inline standalone_app/src/components/PlotImportance.tsx:70]: How about using `<Fragment>` instead?
https://react.dev/reference/react/Fragment

```suggestion
    <>
      <Typography
        variant="h6"
        sx={{ margin: "1em 0", fontWeight: theme.typography.fontWeightBold }}
      >
        Hyperparameter Importance
      </Typography>
      <Box id={plotDomId} sx={{ height: "450px" }} />
    </>
```

File: standalone_app/src/components/PlotIntermediateValues.tsx [hunk 2]
```diff
@@ -23,17 +23,15 @@ export const PlotIntermediateValues: FC<{
...
-    </Card>
+    <Box>
+      <Typography
+        variant="h6"
+        sx={{ margin: "1em 0", fontWeight: theme.typography.fontWeightBold }}
+      >
+        Intermediate values
+      </Typography>
+      <Box id={plotDomId} sx={{ height: "450px" }} />
+    </Box>
```
Comment (c-bata) [inline standalone_app/src/components/PlotIntermediateValues.tsx:34]: ditto.

```suggestion
    <>
      <Typography
        variant="h6"
        sx={{ margin: "1em 0", fontWeight: theme.typography.fontWeightBold }}
      >
        Intermediate values
      </Typography>
      <Box id={plotDomId} sx={{ height: "450px" }} />
    </>
```

**Review by c-bata — COMMENTED** [review]
Good catch. Changes look good to me. I left one minor suggestion.

**Comment by keisuke-umezawa** [comment]
@c-bata 
Thank you for reviewing. I merged your suggestions.

**Pull request merged**